indico/modules/events/management/forms.py

```diff
@@ -168,6 +168,13 @@ def validate_end_dt(self, field):
 
 class EventLocationForm(IndicoForm):
     location_data = IndicoLocationField(_('Location'), allow_location_inheritance=False)
+    own_map_url = URLField(_('Map URL'), [URL()])
```

Comment:
You need `[Optional(), URL()]` or an empty field will fail validation.

Reply:
Done! (So easy to forget all the quirks...)